Chat/Defines.h

```diff
@@ -20,7 +20,7 @@
 //1 for enabling
 #define BROADCAST_LOOPBACK 0
 
-
```

Comment:
Well, you shouldn't commit such changes. Use git diff to see all files that contains changes. If there are only blank line insertion, use git checkout -- <filename> to revert current file.